api/cmd/auth/auth.go

```diff
@@ -0,0 +1,95 @@
+package main
+
+import (
+	"fmt"
+
+	"net/http"
+	"os"
+
+	"github.com/gorilla/handlers"
+	"github.com/gorilla/mux"
+	"github.com/gorilla/sessions"
+	"github.com/markbates/goth"
+	"github.com/markbates/goth/gothic"
+	"github.com/markbates/goth/providers/bitbucket"
+	"github.com/markbates/goth/providers/github"
+	"github.com/markbates/goth/providers/gitlab"
+	"github.com/tektoncd/hub/api/pkg/app"
+	auth "github.com/tektoncd/hub/api/pkg/auth"
+)
+
+func main() {
+
+	key := ""            // Replace with your SESSION_SECRET or similar
+	maxAge := 86400 * 30 // 30 days
+	isProd := false      // Set to true when serving over https
+
+	store := sessions.NewCookieStore([]byte(key))
+	store.MaxAge(maxAge)
+	store.Options.Path = "/"
+	store.Options.HttpOnly = true // HttpOnly should always be enabled
+	store.Options.Secure = isProd
+
+	gothic.Store = store
```

Comment:
For now we are adding this to avoid the comment from the goth library while running the auth server